src/js/core.js

```diff
@@ -2803,8 +2803,7 @@
     // Try to detect mobile devices
     // ============================
 
-    isMobile:
-      document.createTouch !== undefined && /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent),
```

Comment:
Alternatively, the feature detect that [Modernizr is currently doing](https://github.com/Modernizr/Modernizr/blob/501cc0a1d761ce1a789e3f776790a05c6f2fd3d8/feature-detects/touchevents.js#L39) could be used instead of removing it completely:

```js
(('ontouchstart' in window) || window.DocumentTouch && document instanceof DocumentTouch)
```